Jonesy22/Perro
# PER-2 create left side component

Moved around the files to allow for npm start to work and added a couple of components for the left side feature. Wanted to merge to master before you guys begin development so it works for everyone.

Test plan:
- Checkout this `PER-2-create-left-side-component` branch
- Run `npm start` and confirm the application is available on localhost:300
